src/neo/IO/Json/JString.cs

```diff
@@ -46,6 +46,11 @@ internal override void Write(Utf8JsonWriter writer)
             writer.WriteStringValue(Value);
         }
 
+        public override JObject Clone()
+        {
+            return new JString(Value);
```

Comment:
`JString` is immutable. So we don't need to create a new instance.

Reply:
Thanks Erik. I didn't know they were immutable. It feels a little weird to have a clone function that returns the same object. The test even failed because now both objects point to the same reference.

Feels weird, but if it is immutable, I think it is ok, I will update it to return 'this' and remove the reference comparison.